src/HasTranslations.php

```diff
@@ -63,6 +67,26 @@ public function translationExists(string $locale): bool
         return $this->translations()->where($this->getLocaleKeyName(), $locale)->exists();
     }
 
+    /**
+     * Get all the translations.
+     *
+     * @return \Illuminate\Database\Eloquent\Collection
+     */
+    public function getTranslations()
+    {
+        return $this->translations()->get();
+    }
+
+    /**
+     * Purge the translations.
+     *
+     * @return mixed
+     */
+    public function purgeTranslations()
+    {
+        return $this->getTranslations()->each->delete();
```

Comment:
Why don't you do `$this->translations()->delete();` here? That'll only do one query, even if there's 1000 translations stored for this model (unlikely, but still 😉).

Reply:
When doing just `->translations()->delete()`, the model events from the Translations will not be called, this prevents that, but it might be unnecessary as there shouldn't be any more logic in the translation model.